src/test/java/core/basesyntax/BalanceHandlerTest.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.operation.BalanceHandler;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+    private static BalanceHandler balanceHandler;
+
+    @Test
+    public void balanceHandlerTest_getOperationAction_OK() {
```

Comment:
Используй одинаковый подход к названию всех тестовых методов. Например можно использовать такой же принцип именования методов, как и в задании registration-validation:
![image](https://user-images.githubusercontent.com/117547096/215284867-42f66e08-83ad-4978-8f6a-4a4234f2e5c1.png)
Имя тестируемого сервиса не нужно указывать в названии метода
```suggestion
    public void getOperationAction_ok() {
```